docs/spec/openapi/public-docs.yaml

```diff
@@ -0,0 +1,5703 @@
+      send traces, metrics, and logs to APM Server. OTLP is the default transfer
+      protocol for OpenTelemetry and is supported natively by APM Server. APM
+      Server supports two OTLP communication protocols on the same port:
+      OTLP/HTTP (protobuf) and OTLP/gRPC.
+    x-displayName: APM OpenTelemetry intake
+  - name: server info
+    description: APIs that query general APM Server information.
+    x-displayName: APM server information
+paths:
+  /:
+    get:
+      summary: Get general server information
+      description: >
+        This lightweight endpoint is useful as a server up/down health check.
+
+        To configure authenticated access to the APM server, the instructions at
+        [APM API
+        key](https://www.elastic.co/guide/en/observability/current/api-key.html)
+        or [APM Secret
+        Token](https://www.elastic.co/guide/en/observability/current/secret-token.html)
+        must be followed to configure the correct permissions for APM access.
+      operationId: getServerHealth
+      tags:
+        - server info
+      responses:
+        '200':
+          $ref: '#/components/responses/200_server_info'
+  /config/v1/agents:
+    get:
+      summary: Get agent configuration changes
+      description: >
+        To configure authenticated access to the APM server, the instructions at
+        [APM API
+        key](https://www.elastic.co/guide/en/observability/current/api-key.html)
+        or [APM Secret
+        Token](https://www.elastic.co/guide/en/observability/current/secret-token.html)
+        must be followed to configure the correct permissions for APM access.
+      operationId: getAgentConfig
+      tags:
+        - agent config
+      parameters:
+        - in: query
+          name: service.name
+          required: true
+          schema:
+            type: string
+        - in: query
+          name: service.environment
+          schema:
+            type: string
+      responses:
+        '200':
+          description: A successful response.
+          content:
+            application/json:
+              schema:
+                type: object
+    post:
+      summary: Get agent configuration changes
+      description: >
+        To configure authenticated access to the APM server, the instructions at
+        [APM API
+        key](https://www.elastic.co/guide/en/observability/current/api-key.html)
+        or [APM Secret
+        Token](https://www.elastic.co/guide/en/observability/current/secret-token.html)
+        must be followed to configure the correct permissions for APM access.
+      operationId: postAgentConfig
+      tags:
+        - agent config
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - service
+              properties:
+                CAPTURE_BODY:
+                  type: string
+                  example: 'off'
```

Comment:
capture_body exists in the response, but I don't find anything about it in the request handling code. The fact that it is all caps makes it look a bit out of place as well.